Library/Formula/sfml.rb

```diff
@@ -27,9 +17,45 @@ class Sfml < Formula
   depends_on "glew"
   depends_on "jpeg"
   depends_on "libsndfile"
+  depends_on "doxygen" => :optional
+
+  # Remove the bundled libs wording that causes linking errors.
+  # https://github.com/Homebrew/homebrew/pull/35279
+  patch :DATA
 
   def install
-    system "cmake", ".", "-DCMAKE_INSTALL_FRAMEWORK_PREFIX=#{frameworks}/", *std_cmake_args
+    args = std_cmake_args
+    args << "."
+
+    args << "-DFREETYPE_LIBRARY=#{Formula["freetype"].opt_lib}/libfreetype.dylib"
+    args << "-DGLEW_LIBRARY=#{Formula["glew"].opt_lib}/libGLEW.dylib"
+    args << "-DGLEW_INCLUDE_PATH=#{Formula["glew"].opt_include}"
+    args << "-DJPEG_LIBRARY=#{Formula["jpeg"].opt_lib}/libjpeg.dylib"
+    args << "-DSNDFILE_LIBRARIES=#{Formula["libsndfile"].opt_lib}/libsndfile.dylib"
+    args << "-DSNDFILE_INCLUDE_DIR=#{Formula["libsndfile"].opt_include}"
```

Comment:
Theoretically, all of this block can be ditched if we remove the ` extlibs ` path as I've done below. Without that it behaves as you'd, well, expect to a larger degree, and goes path sniffing instead:

```
-- Found GLEW: /usr/local/lib/libGLEW.dylib
-- Found JPEG: /usr/local/lib/libjpeg.dylib
-- Found Freetype: /usr/local/lib/libfreetype.dylib (found version "2.5.5")
-- Found OpenAL: /usr/local/lib/libopenal.dylib
-- Found SNDFILE: /usr/local/lib/libsndfile.dylib
==> Summary
🍺  /usr/local/Cellar/sfml/2.2: 103 files, 1.7M, built in 81 seconds
```

I was taking prior suggestion from upstream here that we be more explicit about it, but I'm happy to burn this whole darn chunk.